edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -469,7 +469,7 @@ def test_get_config(self):
         headers = self.build_jwt_headers(nonstaff_user)
         response = self.client.get(self.url, **headers)
         self.assertEqual(200, response.status_code)
-        self.assertEqual('test_provider', response.data.get('provider'))
```

Comment:
Why is the `.get` removed here?

Reply:
Ah right, I was trying to put all of my response.data.get('var') calls into a response.data['var'] format but it looks like that got caught along the way, I can put it back if you'd like.